encoding/test/main.go

```diff
@@ -49,30 +50,55 @@ func readpoints() {
 }
 
 func TestKzgRs() {
-	numSymbols := 3
+	isSmallTest := false
+
+	numSymbols := 4096 / 8
 	// encode parameters
-	numNode := uint64(4) // 200
-	numSys := uint64(2)  // 180
+	numNode := uint64(4096) // 200
+	numSys := uint64(512)   // 180
 	numPar := numNode - numSys
-	// Prepare data
-	fmt.Printf("* Task Starts\n")
-	fmt.Printf("    Num Sys: %v\n", numSys)
-	fmt.Printf("    Num Par: %v\n", numPar)
-	//fmt.Printf("    Data size(byte): %v\n", len(inputBytes))
+
+	numDevices, _ := cr.GetDeviceCount()
+	fmt.Println("num device ", numDevices)
```

Comment:
This function is just a scratch pad right? you're not using numDevices anywhere and you even have `p.UseGpu = false` below. What are you testing here exactly?